docs/api-reference/carto/basemap.md

```diff
@@ -2,10 +2,14 @@
 
 [CARTO basemaps](https://carto.com/basemaps/) are available and they can be used without a token.
 
-Ensure you follow the [Terms and Conditions](https://drive.google.com/file/d/0B3OBExqwT6KJNHp3U3VUamx6U1U/view) when using them.
+Ensure you follow the [Terms and Conditions](https://drive.google.com/file/d/1P7bhSE-N9iegI398QYDjKeVhnbS7-Ilk/view) when using them.
 
 
-## Usage
+## React
```

Comment:
I'd like to keep the usage section and have a subsection for React and Standalone

Reply:
Cool, fixed in latest commit.